xbmc/addons/kodi-addon-dev-kit/include/kodi/addon-instance/Inputstream.h

```diff
@@ -155,6 +159,14 @@ extern "C" {
     CRYPTO_INFO m_cryptoInfo;
   } INPUTSTREAM_INFO;
 
+  typedef struct INPUTSTREAM_TIMES
```

Comment:
why is that a typedef?

Reply:
cp/paste from alwins coding style, whats wrong with that?

Reply:
nothing really wrong but imo there is no advantage. I always remove the typedef when I hit those.